Book.java

```diff
@@ -0,0 +1,32 @@
+public class Book {
+    Book(int initialNoOfBook){
+        this.noOfCopies = initialNoOfBook;
+    }
+    private int noOfCopies ;
```

Comment:
follow coding standard and keep the member variables before the constructor or any other methods.